python/docs/linked_list_implementation/tests/test_linked_list_insertions.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+from linked_list.linked_list import LinkedList, TargetError
```

Comment:
Pytest currently is generating this error:

`ModuleNotFoundError: No module named 'linked_list'`

The fix is to change the import to this:

`from data_structures.linked_list import LinkedList, TargetError`